Rocket.Chat/Models/AuthSettings.swift

```diff
@@ -30,6 +30,17 @@ final class AuthSettings: BaseModel {
     @objc dynamic var isGoogleAuthenticationEnabled = false
     @objc dynamic var isLDAPAuthenticationEnabled = false
 
+    // Registration
+    enum RegistrationFormAccess: String {
+        case isPublic = "Public", isDisabled = "Disabled", isSecretURL = "Secret URL"
```

Comment:
1. Break each case per line, please
2. Move the enum to outside of the class, please (top)

Reply:
Alright.
Not sure about making that enum global but that's your call.